contracts/AnchoredView/AnchoredView.sol

```diff
@@ -190,50 +178,50 @@ contract AnchoredView is SymbolConfiguration {
      * @param cToken The cToken address for price retrieval
      * @return The price for the given cToken address
      */
-    function getUnderlyingPrice(address cToken) external view returns (uint256) {
+    function getUnderlyingPrice(address cToken) public returns (uint) {
+        CTokenMetadata memory tokenConfig = getCTokenConfig(cToken);
         if (breaker == true) {
-            return readAnchor(cToken);
+            return readAnchor(tokenConfig);
```

Comment:
should it be have the same adjustment of precision as in the last case : `mul(usdPerEth, ethPerToken) / 1e6;` ?

Reply:
up for discussion, but seems like falling back all the way is winning the day right now. Just pretend this whole thing never happened and serve prices in ether from v1 oracle